### PR TITLE
384120 Update ADP namespace & owner

### PR DESCRIPTION
**Pull request by rtasalem**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/384120

### CODE REVIEW

**Comment by rtasalem** [comment]
Originally this PR was for setting up the frontend for the customer queries technical spike, however this has been put on hold to be picked up at a later date. Ended up using this PR to instead update the `namespace`, `owner` and `workstream` from `fcp` to `ffc`, but ADP confirmed that all of our services are migrating to the new `fcp` namspace so have undone those changes. Only change this PR now makes is adding back in the `programmeName` parameter.

File: catalog-info.yaml [hunk 1]
```diff
@@ -40,5 +40,5 @@ metadata:
 spec:
   type: frontend
   lifecycle: alpha
-  owner: "group:default/fcp-sfd"
+  owner: "group:default/fpc-sfd"
```
Comment (simonnineteen90) [inline catalog-info.yaml]: Is this correct? `fpc` ?

Comment (rtasalem) [reply catalog-info.yaml]: It is definitely not correct... will change this now, thank you

Comment (rtasalem) [reply catalog-info.yaml]: Done

**Review by simonnineteen90 — APPROVED** [review]

**Pull request merged**